src/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
    "author": "Jonathan Peppers, LanguageTooler GmbH",
    "background": {
-      "scripts": [ "packed.js", "common/browser-polyfill.min.js", "config/config.js", "common/browserDetector.js", "common/utils.js", "common/eventBus.js", "common/messages.js", "common/i18nManager.js", "common/extensioni18nManager.js", "common/languageManager.js", "common/storageController.js", "common/extensionStorageController.js", "common/environmentAdapter.js", "common/extensionEnvironmentAdapter.js", "common/extension-init.js", "common/tracker.js", "background/graphemeSplitter.js", "background/validator.js", "background/dictionarySync.js", "background/synonyms.js", "background/extension-main.js" ]
+      "service_worker": "packed.js"
```

Comment:
This can't be an array anymore:

https://stackoverflow.com/a/66055883

So I tried to pack all these files into one.